lib/Notes/NotesSmartAccordion/NotesSmartAccordion.js

```diff
@@ -102,7 +111,11 @@ class NotesSmartAccordion extends Component {
     sortParams: {
       order: ASC,
       by: assigningModalColumnsConfig.names.TITLE,
-    }
+    },
+    sortedColumn: {
+      column: notesColumnNames.DATE,
+      isDesc: false,
+    },
```

Comment:
Can we operate directly on the `sortParams` state field? For me, it's very confusing to see both `sortParams` and `sortedColumn`.  It's difficult to understand what `sortedColumn` is responsible for.

Reply:
No, `sortParams` and `sortedColumn` has different parms and used in different places. `sortParams` used for search assigning modal and `sortColumn` used for sorting columns in notes list

Reply:
@Vladyslav-Velytskyi , oh I see, thank you!